docs/src/components/GuidelinesBuilder/GuidelinesPageBuilder.js

```diff
@@ -0,0 +1,169 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+import BpkContentContainer from 'bpk-component-content-container';
+import { cssModules } from 'bpk-react-utils';
+import ChevronRightIcon from 'bpk-component-icon/lg/chevron-right';
+import { withRtlSupport } from 'bpk-component-icon';
+
+import HeroSection from '../HeroSection';
+import Heading from '../Heading';
+
+import STYLES from './GuidelinesPageBuilder.scss';
+
+const RtlSupportedChevron = withRtlSupport(ChevronRightIcon);
+const getClassName = cssModules(STYLES);
+
+type SectionProps = {
+  id: string,
+  title: string,
+  content: Node,
+  alternative: boolean,
+};
+
+type GuidelinesPageBuilderProps = {
+  hero: any,
+  sections: Array<SectionType>,
+  title: string,
+  nextPageLink: any,
+};
+
+type NavigationFooterProps = {
+  id: string,
+  title: string,
+  subtitle: string,
+  link: string,
+};
+
+const Section = (props: SectionProps) => {
+  const { id, title, content, alternative } = props;
+  const sectionClassName = [
+    getClassName('bpk-docs-guidelines-page__section'),
+    alternative &&
```

Comment:
can we call this `alternate` instead of `alternative`. Alternate is more clear about what it does

Reply:
All done 😄